<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Associate Detections</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon/cube1.svg</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="vert_layout">
    <item>
     <widget class="QLabel" name="image_label">
      <property name="maximumSize">
       <size>
        <width>9999999</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="CustomLabel" name="image_view">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenDetections"/>
    <addaction name="actionSave"/>
    <addaction name="actionRestart"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenDetections"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionRestart"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Camera">
   <property name="windowTitle">
    <string>Objects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="add_object_button">
       <property name="text">
        <string>Add new Object</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="objects_list"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../assets/resources.qrc">
     <normaloff>:/icon/folder-open-solid.svg</normaloff>:/icon/folder-open-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Open Associations Dataset</string>
   </property>
   <property name="toolTip">
    <string>Open an associations dataset</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../assets/resources.qrc">
     <normaloff>:/icon/save-solid.svg</normaloff>:/icon/save-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save Bounding Box Dataset&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset resource="../assets/resources.qrc">
     <normaloff>:/icon/redo-alt-solid.svg</normaloff>:/icon/redo-alt-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="toolTip">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpenDetections">
   <property name="icon">
    <iconset resource="../assets/resources.qrc">
     <normaloff>:/icon/image-open.svg</normaloff>:/icon/image-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open Detections</string>
   </property>
   <property name="toolTip">
    <string>Open images and detections files</string>
   </property>
  </action>
  <zorder>dockWidget_Camera</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>customLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
